src/LaravelPassport/Provider.php

```diff
@@ -104,7 +104,7 @@ protected function getLaravelPassportUrl($type)
         return rtrim($this->getConfig('host'), '/').'/'.ltrim($this->getConfig($type, Arr::get([
             'authorize_uri' => 'oauth/authorize',
             'token_uri'     => 'oauth/token',
-            'userinfo_uri'  => 'api/user',
+            'userinfo_uri'  => $this->getConfig('userinfo_uri')',
```

Comment:
Can you add the default value? otherwise it will be null if you don't set that config.